pkgs/applications/system/zxfer/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, fetchFromGitHub
+, installShellFiles
+, makeWrapper
+, coreutils
+, gawk
+, gnugrep
+, gnused
+, rsync
+, stdenv
+, zfs
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zxfer";
+  version = "1.1.7";
+
+  src = fetchFromGitHub {
+    owner = "allanjude";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-11SQJcD3GqPYBIgaycyKkc62/diVKPuuj2Or97j+NZY=";
+  };
+
+  nativeBuildInputs = [ installShellFiles makeWrapper ];
+
+  installPhase = ''
+    installManPage zxfer.1m zxfer.8
+    install -Dm755 ${pname} -t $out/bin/
+    wrapProgram $out/bin/${pname} --prefix PATH : ${lib.makeBinPath [
+      coreutils
+      gawk
+      gnugrep
+      gnused
```

Comment:
```suggestion
      gawk
```

Reply:
> waiting on consensus from https://github.com/NixOS/nixpkgs/pull/208333#discussion_r1059488315

Reply:
Its not in the default installed tools

Reply:
can you point me to a listing of these "base installed tools"?

Reply:
These things are usually listed around many NixOS modules...

Reply:
I have stolen the approach from #208333, so this is probably moot, but I would be keen to get a link to those tools in #209374 as I do not know what to look for in `./nixos`